recipes/leftbar/general_row/leftbar_general_row_icon.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <dt-icon
+    v-if="shouldRenderWithIcon"
+    :name="getIconName"
+    :class="iconColor"
+    size="300"
+  />
+  <div
+    v-else-if="isContactCenterType"
+    :class="contactCenterIconClasses"
+  />
+  <div
+    v-else-if="isDialbotType"
+    :class="dialbotClasses"
+  >
+    <icon-dialpad-glyph
+      class="d-svg--size16"
+      color="white"
+    />
```

Comment:
Used the v6 icon here as it's not direct mapping in the dialtone-icons v7

Reply:
Yeah closest we have is the dialpad brand icon from https://dialpad.design/design/icons/. I wouldn't worry about this at the moment.